### ISSUE TITLE
Unused ModelBuilder/ModelBatch call?

### ISSUE THREAD

**Issue by SevenDayCandle** [SrJohnathan/gdx-effekseer#2]
In EffekseerManager.java (https://github.com/SrJohnathan/ParticlesLIbgdx/blob/4743bac38588c525f021a9bf5c0af701d54facc9/src/main/java/br/com/johnathan/gdx/effekseer/api/EffekseerManager.java), I noticed that there are two variables that don't seem to be used: mb and modelbatch (as well as the function that call/set them. Were these meant to be used for something in the future, or are these leftovers from an earlier version of the code?

**Comment by SrJohnathan** [comment]
Yes, it was collision detection code that I need to finish, I don't have time to factor this code, because of my company, it will take a while to fix it, I'm developing a water lib for libgdx too
